### PR TITLE
AFDP-1678: Awards module - Tree view doesn't work

### PR DESCRIPTION
**Pull request by BukicMicko**
With lates changes for ArkCase core application, Awards module was not workin. This PR fix that.

### CODE REVIEW

File: gms-user-interface/src/main/resources/META-INF/resources/resources/modules/award/controllers/award-list.client.controller.js [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+                    nodeId: Util.goodValue(caseInfo.id, 0)
+                    , nodeType: ObjectService.ObjectTypes.CASE_FILE
+                    , nodeTitle: Util.goodValue(caseInfo.title)
+                    , nodeToolTip: Util.goodValue(caseInfo.title)
+                };
+            }
         });
 
-        var treeHelper = new HelperObjectTreeService.Tree({
+        /*var treeHelper = new HelperObjectTreeService.Tree({
```
Comment (ristetutureski) [inline gms-user-interface/src/main/resources/META-INF/resources/resources/modules/award/controllers/award-list.client.controller.js]: Are we going to keep this commented code, or it's safe to remove it?

File: gms-user-interface/src/main/resources/META-INF/resources/resources/modules/award/controllers/award.client.controller.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+            , stateParams: $stateParams
+            , moduleId: "award"
+            , getObjectInfo: CaseInfoService.getCaseInfo
+            , updateObjectInfo: CaseInfoService.updateCaseInfo
+            , initComponentLinks: function (config) {
+                return HelperObjectBrowserService.createComponentLinks(config, ObjectService.ObjectTypes.CASE_FILE);
+            }
+        });
+
+        /*var promiseGetModuleConfig = ConfigService.getModuleConfig("award").then(function (config) {
```
Comment (ristetutureski) [inline gms-user-interface/src/main/resources/META-INF/resources/resources/modules/award/controllers/award.client.controller.js]: Here the same ... can we remove it, or we needed it?

**Comment by ristetutureski** [comment]
:+1: Because Manoj is on vacation, I will merge this PR. For me looks good.

File: gms-user-interface/src/main/resources/META-INF/resources/resources/modules/award/controllers/award-list.client.controller.js (hunk 1 above)
Comment (BukicMicko) [inline gms-user-interface/src/main/resources/META-INF/resources/resources/modules/award/controllers/award-list.client.controller.js]: I was keeping it while testing, but i don't need it anymore. I will remove it in some of the next PRs.

**Pull request merged**